qemu/tests/blockdev_commit_reboot.py

```diff
@@ -0,0 +1,63 @@
+from virttest import utils_disk
+
+from provider import backup_utils
+from provider import job_utils
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitReboot(BlockDevCommitTest):
+
+    def configure_data_disk(self, tag):
+        os_type = self.params["os_type"]
```

Comment:
No need to get path, all you want is a dir like /tmp for linux, or C: for windows, so, just fill disks_info with that information only. 
It's not a good idea to rewrite configure_data_disk here, for what you want is  the system image, so why not write a method in base class?

Reply:
Have update， thanks